### PR TITLE
Added Navigation to Course Detail Page

### LINKED ISSUE #144: Course Details Page: Navigation Functionality
## Describe the bug

There is no navigation functionality on the Course Details Page.

## To Reproduce

1. Open app, go to /course/:id route
2. Navigation Buttons do not have any functionality 

## Expected behavior

Navigation Buttons should do proper functions like back button redirects to homepage.

## Environment (please complete the following information):

- OS: Pop!_OS
- Browser: Firefox Nightly

### PR DESCRIPTION
**Pull request by rizwan2000rm**
**Issue Number**

fixes #144 

**Describe the changes you've made**

Added back to home functionality to topbar on course detail page. Also added a menu for drawer button.

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
Please provide screenshots of the changes

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 A click on Back button takes you back to /home and options opens a menu component from Material UI.
Navigation

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 Resolved merge conflicts. Does UI need any changes I'm using the Menu from MaterialUI. It closes on clicking outside the menu container.

File: src/pages/CourseDetail/index.js [hunk 1]
```diff
@@ -56,14 +59,36 @@ const coursesData = courseDetail[0].coursesSection;
...
+    setAnchorEl(null);
+  };
+
   return (
     <>
       <div id="back-to-top-anchor"></div>
       <ScrollToTop />
       <div className={classes.navigation}>
-        <ArrowBackIcon className={classes.svg} onClick={() => history.goBack()} />
-        <MoreVertIcon className={classes.svg} />
```
Comment (Abhishek-kumar09) [inline src/pages/CourseDetail/index.js:66]: To me `history.goBack()` was  much better than always routing it to `/`.
 You can revert that code or if there was any problem with the previous code you can tell me that too.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Changes are good, some minor suggestions

File: src/pages/CourseDetail/index.js (hunk 1 above)
Comment (rizwan2000rm) [reply src/pages/CourseDetail/index.js:66]: Will revert this. Yeah actually go back makes more sense as we have swipeable routes.

**Comment by rizwan2000rm** [comment]
@Abhishek-kumar09 Reverted requested changes.

**Comment by Abhishek-kumar09** [comment]
Thanks @rizwan2000rm

**Pull request merged**